Debian.md

```diff
@@ -11,16 +11,20 @@ All packages we provide are signed with the following [key](https://packages.ici
 
 Here’s how to add the official release repository:
 
+(Note: the step with `mkdir /etc/apt/keyrings ...` is only necessary on older systems where this directory is not created by default)
```

Comment:
I guess the suggested `install` part wouldn't need this comment.

Reply:
Indeed, `install` does not throw an error if the directory already exists.

Reply:
Removed it.